source-build.slnf

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
we need to keep updating this file whenever there is a new project created? What if we forget?

Reply:
@MichaelSimons ?

Reply:
If you forget, the new project will not be included in source build outputs. If the new project is not _used_ by source build consumers, then there is no problem. If the project is used, either the build will fail early validation, or someone will eventually ask for it and it gets added.